### PR TITLE
Wrap tooltip label if too long

### PR DESCRIPTION
**Pull request by philipperolet**
Assistant descriptions, for instance, can be lengthy

### CODE REVIEW

File: sparkle/src/components/Tooltip.tsx [hunk 1]
```diff
@@ -47,7 +47,9 @@ export function Tooltip({
   const tooltipPositionClasses =
     position === "above" ? "s-bottom-full s-mb-2" : "s-top-full s-mt-2";
 
-  const labelClasses = "s-whitespace-nowrap";
+  // if tooltip text is too long we need to wrap it
+  const labelLength = label?.length || 0;
+  const labelClasses = labelLength > 80 ? "w-[38em]" : "s-whitespace-nowrap";
```
Comment (spolu) [inline sparkle/src/components/Tooltip.tsx:52]: I know we can use a tailwind system length that fits here right? 😬

Comment (philipperolet) [reply sparkle/src/components/Tooltip.tsx:52]: I had looked, max one is 96 which is 24rem not enough; and we don't want relative width either but rather ~80 character width

Comment (spolu) [reply sparkle/src/components/Tooltip.tsx:52]: 96 is large enough no?

**Comment by spolu** [comment]
Deferring review to @Duncid

**Review by Duncid — APPROVED** [review]

File: sparkle/src/components/Tooltip.tsx (hunk 1 above)
Comment (philipperolet) [reply sparkle/src/components/Tooltip.tsx:52]: I thought about this in the context of potentially multiparagraph long descriptions (assistant descriptions). I had heard of 70-80 chars/line being the optimal for reading, with a max at 100 and a min ~ 60

w-96 is ~ 50 chars. I'm not strongly opinionated on this so ready to change depending on your opinion @Duncid

Comment (spolu) [reply sparkle/src/components/Tooltip.tsx:52]: @Duncid approved :+1: good to go :+1:

**Pull request merged**